G3D10/G3D-app.lib/include/G3D-app/DefaultRenderer.h

```diff
@@ -123,12 +208,11 @@ class DefaultRenderer : public Renderer {
     /** Subclasses that can compute global illumination to deferred shading buffers should override this method,
         which is invoked before renderDeferredShading. */
     virtual void renderIndirectIllumination
-       (RenderDevice*                       rd,
-        const Array<shared_ptr<Surface> >&  sortedVisibleSurfaceArray, 
-        const shared_ptr<GBuffer>&          gbuffer, 
-        const LightingEnvironment&          environment) {
-        debugAssertM(m_deferredShading, "Renderer::renderIndirectIllumination should only be invoked when in deferred shading mode");
-    }
+    (RenderDevice* rd,
+        const Array<shared_ptr<Surface> >& sortedVisibleSurfaceArray,
+        const shared_ptr<GBuffer>& gbuffer,
+        const LightingEnvironment& environment,
+        const shared_ptr<TriTree>& tritree = nullptr);
```

Comment:
I notice this adds a parameter to `renderIndirectIllumination()`. It seems like nothing in the solution was using it other than the `DefaultRenderer` so I think this change is fine.